CONTRIBUTING.md

```diff
@@ -3,7 +3,7 @@
 ## Installation
 
 * `git clone <repository-url>`
-* `cd ember-mapbox-composer`
+* `cd nycplanning-ember`
```

Comment:
```suggestion
* `cd ose-ember-addons`
```
I'm assuming we're keeping the same repo name and that contribution is through this repo. In which case, I believe the folder will be ose-ember-addons